modules/core/include/opencv2/core/utility.hpp

```diff
@@ -773,7 +773,7 @@ The sample below demonstrates how to use CommandLineParser:
 The keys parameter is a string containing several blocks, each one is enclosed in curly braces and
 describes one argument. Each argument contains three parts separated by the `|` symbol:
 
--# argument names is a space-separated list of option synonyms (to mark argument as positional, prefix it with the `@` symbol)
+-# argument names is a list of option synonyms separated by ASCII space characters (0x20) (to mark argument as positional, prefix it with the `@` symbol)
```

Comment:
It's too strong statement, as there is no `0x20` constant in the code. The value comes from here: https://github.com/opencv/opencv/blob/0b6c9a2123a8ff1fc0d9e340c8d0132dac3edb6b/modules/core/src/command_line_parser.cpp#L46
It's just space `' '` symbol in source code. I propose to re-write it more generic:  "argument names is a list of option synonyms separated by standard space characters".

Reply:
Thank you for pointing this out. I will revise as you suggested.